core/modules/filters/is.js

```diff
@@ -27,40 +27,44 @@ Export our filter function
 */
 exports.is = function(source,operator,options) {
 
+	var isFilterOperators = getIsFilterOperators(),
+		subops = operator.operand.split("+");
 
-	if( !operator.operand) {
-		// Return all tiddlers if the operand is missing
+	//Make sure all the operands are defined.
+	for (var t = 0; t < subops.length; t++){
+		if( !isFilterOperators[subops[t]] ) {
+			return [$tw.language.getString("Error/IsFilterOperator")];
+		}
+	}
+
+
+	if(subops.length === 0) { 	// Return all tiddlers if the operand is missing
 		var results = [];
 		source(function(tiddler,title) {
 			results.push(title);
 		});
 		return results;
-	}
+	} else if(subops.length === 1) {	// Shortcut the Single Operator
+		var subop = isFilterOperators[subops[0]];
```

Comment:
`subop` is very close to `subops` ... So I was confused the very first time.... IMO it should be a variable name with is a bit more difference.